app.py

```diff
@@ -33,6 +50,27 @@ def create_index(company: str, authorization: str = Header(None, convert_undersc
     is_authorized_request(auth=authorization)
     build_pinecone_index(company)
 
+@app.post("/api/invoice")
+def upload_invoice(file: UploadFile = File(...),authorization: str = Header(None, convert_underscores=True)):
+    is_authorized_request(auth=authorization)
+    file_path = UPLOAD_DIR / file.filename
+    with open(file_path, "wb") as buffer:
+        shutil.copyfileobj(file.file, buffer)
+    file_type=get_file_type(file.filename)
```

Comment:
please give proper spacing in code for better readability

Reply:
Ok!

Reply:
i will add comments as well!